server/controllers/comment.controller.js

```diff
@@ -126,6 +126,33 @@ class CommentController {
     };
     return Response(res, 201, 'Comment liked successfully', likeObject);
   }
-}
 
+  /**
+   * Edit comment controller
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} response
+   */
+  static async editComment(req, res) {
+    const { id } = req.params;
+    const userId = req.user.id;
+    const { comment } = req.body;
+
+    const editedComment = await Comment.update({
+      comment
+    }, {
+      returning: true,
+      where: {
+        userId,
+        id
+      }
+    });
+    if (!editedComment) {
+      return Response(res, 400, 'Comment not found');
+    }
+    if (editedComment) {
```

Comment:
The `if` statement is not needed on line 153 because the controller won't be called except there is a comment.